### PR TITLE
added person implementation

### PR DESCRIPTION
**Pull request by jithingeorgeofl**
Added person classes

### CODE REVIEW

File: src/person/Gender.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package person;
+
+/**
+ * 
+ *
```
Comment (iamjosephmj) [inline src/person/Gender.java]: Remove unused spaces and add documentation to this enum class

**Comment by iamjosephmj** [comment]
@jithingeorgeofl please add unit test to the person class with "junit"

File: src/person/Gender.java (hunk 1 above)
Comment (jithingeorgeofl) [reply src/person/Gender.java]: The issue that you have raised has been resolved

**Comment by iamjosephmj** [comment]
@jithingeorgeofl you can merge this to develop and raise the junit as a different PR

**Pull request merged**